core/scss/utilities/mixins/accessibility/_skiplinks.scss

```diff
@@ -0,0 +1,57 @@
+@charset "UTF-8";
+
+//
+// @skiplinks
+//
+// For providing a link at the top of the page which jumps
+// the user down to an anchor or target at the beginning of the main content.
+//
+// Gives screen readers and keyboard users the same capability of going directly
+// to the main content as sighted, mouse users.
+// of the main content. Learn more at https://webaim.org/techniques/skipnav.
+//
+// Style guide: Mixins.Accessibility.skiplinks
+//
+@mixin skiplinks {
+  @media print {
+    display: none;
+  }
+
+  @include padding(0);
+  background-color: $color-black;
+  color: $color-white;
+  font-family: $font-sans;
+  font-size: .75em;
+  font-weight: 400;
+  left: .8em;
+  min-height: 1px;
+  position: absolute;
+  text-decoration: none;
+  top: -500px;
+  transition-duration: .25s;
+  transition-property: top;
+  transition-timing-function: ease-in-out;
+
+  &,
+  &:hover,
+  &:visited {
+    color: $color-white;
+    height: 1px;
+    overflow: hidden;
+    white-space: nowrap;
+    width: 1px;
+  }
+
+  &:active,
+  &:focus {
+    @include padding(.4em .8em);
+    border: 1px solid $color-cool-grey;
+    border-radius: $border-radius;
+    height: auto;
+    left: .8em;
+    position: fixed;
+    top: .8em;
+    width: auto;
+    z-index: 9999;
+  }
+}
```

Comment:
Why doesn't this extend the helper classes, `.su-sr-only-element` or `.su-sr-only-text`, or include the mixins, `hide-visually`, `hide-text`?

Reply:
It can. The approach here, though, is a little bit different than that of what those two mixins (or related classes) do. I used Homesite as an example of an approach and aimed to mirror that implementation, which only "hides" something from print, but does not leverage nor extend the Bourbon mixins (hiding the element or the text).

While `@include hide-visually;` and `@include hide-visually("unhide");` for hover and focus states might get us something similar as far as behavior, it would be a slight shift from what Homesite is doing, I believe.

Related: I noticed that Homesite had a `_placeholders.scss` file that has a primary purpose of being available for extended code. If we do want to use extends here (or in the future), we might want to consider doing something similar.
https://code.stanford.edu/ucomm/homesite-2017/blob/master/wp-content/themes/homesite17/scss/base/_placeholders.scss

Reply:
Do you know specifically what would be different if we used `hide-visually`? Is it the animation of having the skip links drop down from `top:-500px`?

I'm ok with this as is. I was just curious why we're not using the utilities that were updated as part of this PR.